enterprise/models.py

```diff
@@ -395,6 +401,26 @@ def serialized(self):
         from enterprise.api.v1 import serializers  # pylint: disable=import-outside-toplevel
         return serializers.EnterpriseCustomerSerializer(self).data
 
+    @property
+    def branding_configuration(self):
+        """
+        Return the customer branding configuration or a default object if none exists.
+        """
+        try:
+            return self._branding_configuration
+        except EnterpriseCustomerBrandingConfiguration.DoesNotExist:
+            default_branding = EnterpriseCustomerBrandingConfiguration(
+                enterprise_customer=self,
+                primary_color=DefaultColors.PRIMARY,
+                secondary_color=DefaultColors.SECONDARY,
+                tertiary_color=DefaultColors.TERTIARY,
+            )
+            return default_branding
+
+    @branding_configuration.setter
+    def branding_configuration(self, obj):
+        self._branding_configuration = obj
```

Comment:
Do we need a `@branding_configuration.deleter` defined as well? Or, are we intentionally OK with not handling that here? I'm curious as using the `@property` decorator with `.setter` and `.deleter` is a new concept to me.

Reply:
I just tested this and deleting a customer with an existing branding config (verified in the DB) happens successfully as a cascading deletion. We don't delete the branding config using the customer model anywhere in the code so a `.deleter` shouldn't be needed